lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -665,6 +667,7 @@ SILPassPipelinePlan::getOnonePassPipeline(const SILOptions &Options) {
   SILPassPipelinePlan P(Options);
 
   P.startPipeline("Mandatory Combines");
+  P.addMandatoryInlining();
```

Comment:
This should not be necessary. Please remove this.

Reply:
[`testOriginalProtocolImpl` doesn't inline `cFuncOriginal` in `mandatory_conditional_compile_out_using_optionals.swift`](https://github.com/apple/swift/blob/master/test/SILOptimizer/mandatory_conditional_compile_out_using_optionals.swift#L73) (among others) when this pass isn't run for `Onone`.